ast/src/main/java/com/graphicsfuzz/common/ast/visitors/StandardVisitor.java

```diff
@@ -314,7 +314,9 @@ public void visitVariableDeclInfo(VariableDeclInfo variableDeclInfo) {
   @Override
   public void visitParameterDecl(ParameterDecl parameterDecl) {
     visitChildFromParent(parameterDecl.getType(), parameterDecl);
-    visitChildFromParent(this::visitArrayInfo, parameterDecl.getArrayInfo(), parameterDecl);
```

Comment:
Could there potentially be a new test for this? I may not quite understand what is happening.

Reply:
Done.